### ISSUE TITLE
Fortran pause is stopping server

### ISSUE THREAD

**Issue by TeriForey** [mattkjames7/PyGeopack#4]
Hi Matt,

I'm running PyGeopack within a web server and it's working really well except that occasionally some of the values are generating an error when the user changes the model choice. Unfortunately instead of being a raised exception that I can capture and feedback to the user - it's occurring within the Fortran code and just kills my server.

The kind of error I'm getting is:
```
   ATTENTION:  THE MODEL IS VALID SUNWARD FROM X=-15 Re ONLY,
               WHILE YOU ARE TRYING TO USE IT AT X=  -6466133.50
PAUSE
To resume execution, type go.  Other input will terminate the job.
```

Obviously I can't have the server crashing when a user makes a bad choice, is there any way these kind of errors could be caught within your library so it protects my server?

Thanks,

Teri

**Comment by mattkjames7** [comment]
Hi Teri,

Sorry about this - I had never come across this error before. I have removed the "pause" statement from the Fortran code so it should continue to run now. I have created a new version of the Python package on PyPI: version 0.2.6. 

I had a play with it and I can see why the warning is there in the T01 model - the trace goes pretty crazy when the starting point is far back in the tail. I started at (-25.0,0.0,0.0) to test it and it traces to distances with exponents of > 20. Interestingly, the same starting position used with the T96 model produces a realistic trace. It might be worth switching models in your code at X~-15 Re if you require traces from those positions.

Cheers!
